### PR TITLE
Additional mouse clean up

### PR DESCRIPTION
**Pull request by AndreasArvidsson**

### CODE REVIEW

File: plugin/mouse/mouse.py [hunk 1]
```diff
@@ -84,13 +90,14 @@ def mouse_sleep():
 
     def copy_mouse_position():
         """Copy the current mouse position coordinates"""
-        position = ctrl.mouse_pos()
-        clip.set_text(repr(position))
+        x = actions.mouse_x()
+        y = actions.mouse_y()
```
Comment (phillco) [inline plugin/mouse/mouse.py:94]:
```suggestion
        x, y = actions.mouse_x(), actions.mouse_y()
```

or
```
x, y = ctrl.mouse_pos()
```

File: plugin/mouse/mouse.py [hunk 2]
```diff
@@ -34,6 +34,12 @@
 )
 
 
+@ctx.action_class("main")
```
Comment (phillco) [inline plugin/mouse/mouse.py:37]: Remind me why we use main again instead of user?

**Review by phillco — APPROVED** [review]
Two comments otherwise looks good to me

File: plugin/mouse/mouse.py (hunk 1 above)
Comment (phillco) [reply plugin/mouse/mouse.py:94]: Could add a mouse_pos() action - I think getting the tuple is useful

Comment (AndreasArvidsson) [reply plugin/mouse/mouse.py:94]: We could. I saw that we're actually calling ctrl.mouse_pos in multiple files. I would could make a separate pr changing all of those?

File: plugin/mouse/mouse.py (hunk 2 above)
Comment (AndreasArvidsson) [reply plugin/mouse/mouse.py:37]: `main` is for talent actions without a prefix. `mouse_click`, `key`, etc

**Pull request merged**